analysis/active_analyses.R

```diff
@@ -58,12 +58,17 @@ outcomes <- c("type 1 diabetes",
               "type 2 diabetes - obesity",
               "type 2 diabetes - no obesity",
               "other or non-specific diabetes",
-              "gestational diabetes")
+              "gestational diabetes",
+              "type 1 diabetes - extended follow up",
+              "type 2 diabetes - extended follow up",
+              "other or non-specific diabetes - extended follow up",
+              "gestational diabetes - extended follow up")
 
 outcome_group <- "diabetes"
 
-outcomes_short <- c("t1dm","t2dm", "t2dm_rec", "t2dm_pre_rec", "t2dm_post_rec", "t2dm_follow", "t2dm_pd","t2dm_pd_no", "t2dm_obes","t2dm_obes_no", "otherdm","gestationaldm")
-outcome_venn <- c(TRUE, TRUE, FALSE, FALSE, FALSE, FALSE, FALSE, FALSE, FALSE, FALSE, FALSE, FALSE)
+outcomes_short <- c("t1dm","t2dm", "t2dm_rec", "t2dm_pre_rec", "t2dm_post_rec", "t2dm_follow", "t2dm_pd","t2dm_pd_no", "t2dm_obes","t2dm_obes_no", "otherdm","gestationaldm",
+                    "t1dm_extended_follow_up","t2dm_extended_follow_up", "otherdm_extended_follow_up","gestationaldm_extended_follow_up")
+outcome_venn <- c(TRUE, TRUE, FALSE, FALSE, FALSE, FALSE, FALSE, FALSE, FALSE, FALSE, FALSE, FALSE, FALSE, FALSE, FALSE, FALSE)
 
 for (i in 1:length(outcomes)) {
```

Comment:
You're setting covid_history to TRUE for the extended follow up analyses - do you want this?

Reply:
I've turned all COVID history analyses off now as none of them have ran anyway